deploy/charts/rook-ceph/templates/resources.yaml

```diff
@@ -1889,6 +1889,13 @@ spec:
                         - IPv6
                       nullable: true
                       type: string
+                    multiClusterService:
+                      description: Enable multiClusterService to export the Services between peer clusters
+                      properties:
+                        enabled:
+                          description: Enable multiClusterService to export the mon and OSD services to peer cluster. Ensure that peer clusters should be connected using an MCS API compatible application, like Globalnet Submariner.
```

Comment:
`mon and OSD services` to `MON and OSD services` ?

Reply:
I've always used `mon` because it's not an acronym. `OSD` is an acronym and gets all caps.